classes/native/src/lib.rs

```diff
@@ -1,7 +1,5 @@
-#[macro_use]
-extern crate neon;
-
 use neon::prelude::*;
+use neon::{class_definition, declare_types, impl_managed, register_module};
```

Comment:
Can we make this change in all of the examples?

Reply:
`use neon::prelude::*;` is more consistent with the docs and it seems easier for beginners to start with. Thoughts on this?

Reply:
They aren't included in the prelude. I opened an issue on neon.

https://github.com/neon-bindings/neon/issues/432

Reply:
Including macros in the prelude is a good idea! I think we can make this change after resolving the issue you just opened.